.README/rules/tag-lines.md

```diff
@@ -2,28 +2,39 @@
 
 Enforces lines (or no lines) between tags.
 
+If you only want lines preceding all tags or after all tags, you can use
+the "any" option along with `startLines` and/or `endLines`.
+
+The "always" or "never" options of this rule should not
+be used with the linebreak-setting options of the `sort-tags` rule as both
+may try to impose a conflicting number of lines.
+
 #### Options
 
 The first option is a single string set to "always", "never", or "any"
 (defaults to "never").
 
 "any" is only useful with `tags` (allowing non-enforcement of lines except
```

Comment:
On line 9 above it is said that this rule `tag-lines` should only use `"any"`. But this line 17 here does not mention that. At the same time, it is not good to repeat what is said above. Would it be proper to move line 5-11 to under here instead, or are you afraid users will miss those important details then?

Reply:
LGTM